impeller/entity/contents/content_context.cc

```diff
@@ -604,4 +606,70 @@ void ContentContext::FlushCommandBuffers() const {
   GetContext()->GetCommandQueue()->Submit(buffers);
 }
 
+void ContentContext::AdditionalSetup() const {
```

Comment:
I realize this method is private, but the name makes me die a little bit.

I like the first line of this method instead:

```
void ContentContext::InitializeCommonlyUsedShadersIfNeeded()
```

Reply:
Done